_chapters/handle-forgot-and-reset-password.md

```diff
@@ -165,8 +164,7 @@ export default function ResetPassword() {
   function renderSuccessMessage() {
     return (
       <div className="success">
-        <Glyphicon glyph="ok" />
```

Comment:
`Glyphicon` is not exported from React Bootstrap anymore. We could include another icon set, but this is simpler.

Reply:
Yup, we are using react-icons now.